assets/src/apps/authoring/components/AdaptivityEditor/ActionMutateEditor.tsx

```diff
@@ -138,11 +152,19 @@ const ActionMutateEditor: React.FC<ActionMutateEditorProps> = (props) => {
       <select
         className="custom-select mr-2 form-control form-control-sm"
         id={`action-mutate-operator-${uuid}`}
-        defaultValue={operator}
+        value={operator}
         onChange={(e) => handleOperatorChange(e)}
       >
         {opOptions.map((option) => (
-          <option key={option.key} value={option.value}>
+          <option
+            key={option.key}
+            value={option.value}
+            style={
+              targetType !== ('1' as unknown) && option.key === 'add'
```

Comment:
maybe instead of hiding the option should do like `opOptions.filter().map()` ?

Reply:
fixed.